src/components/page/blocks/BlocksColumn.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-cycle
```

Comment:
moving the reference content into a `ReferenceBlock` that is imported by the `Block` creates a cyclical dependency warning since modals display their content using the `BlockColumn` component.

it's not a true cyclical dependency since modals use a stripped down version of the block layout that can only display content, images, video, or tables so there is no way to load an additional ReferenceBlock and create a true circular dependency i.e. the instance of BlockColumn created by ReferenceBlock can't created a ReferenceBlock.